plugins/mongodb/mongo_conn

```diff
@@ -1,6 +1,22 @@
-#!/usr/bin/python
+#!/usr/bin/env python
+"""
+=head1 NAME
+    MongoDB connections Plugin
 
-## GENERATED FILE - DO NOT EDIT
+=head1 APPLICABLE SYSTEMS
+
+    Works until MongoDB 3.6. The httpinterface was later removed.
```

Comment:
Is there a way to retrieve the values for this plugin in a different way for never versions of mongodb?